src/main/java/com/stripe/model/BankAccount.java

```diff
@@ -1,66 +1,301 @@
    */
-  @Override
-  public BankAccount update(Map<String, Object> params) throws StripeException {
-    return update(params, null);
+  public BankAccount delete(Map<String, Object> params) throws StripeException {
+    return delete(params, (RequestOptions) null);
   }
 
   /**
-   * Update a bank account.
+   * Delete a specified external account for a given account.
+   *
+   * <p>Delete a specified source for a given customer.
    */
-  @Override
-  public BankAccount update(Map<String, Object> params, RequestOptions options)
+  public BankAccount delete(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url;
+    if (this.getAccount() != null) {
+      url =
+          String.format(
+              "%s%s",
+              Stripe.getApiBase(),
+              String.format(
+                  "/v1/accounts/%s/external_accounts/%s", this.getAccount(), this.getId()));
+    } else if (this.getCustomer() != null) {
+      url =
+          String.format(
+              "%s%s",
+              Stripe.getApiBase(),
+              String.format("/v1/customers/%s/sources/%s", this.getCustomer(), this.getId()));
+    } else {
+      throw new InvalidRequestException(
+          "Unable to construct url because [account, customer] field(s) are all null",
+          null,
+          null,
+          null,
+          0,
+          null);
+    }
+    return request(ApiResource.RequestMethod.DELETE, url, params, BankAccount.class, options);
+  }
+
+  /** Verify a specified bank account for a given customer. */
+  public BankAccount verify() throws StripeException {
+    return verify((Map<String, Object>) null, (RequestOptions) null);
+  }
+
+  /** Verify a specified bank account for a given customer. */
+  public BankAccount verify(RequestOptions options) throws StripeException {
+    return verify((Map<String, Object>) null, options);
+  }
+
+  /** Verify a specified bank account for a given customer. */
+  public BankAccount verify(Map<String, Object> params) throws StripeException {
+    return verify(params, (RequestOptions) null);
+  }
+
+  /** Verify a specified bank account for a given customer. */
+  public BankAccount verify(Map<String, Object> params, RequestOptions options)
       throws StripeException {
-    return request(RequestMethod.POST, this.getInstanceUrl(), params, BankAccount.class, options);
+    String url;
+    if (this.getCustomer() != null) {
+      url =
+          String.format(
+              "%s%s",
+              Stripe.getApiBase(),
+              String.format(
+                  "/v1/customers/%s/sources/%s/verify", this.getCustomer(), this.getId()));
+    } else {
+      throw new InvalidRequestException(
+          "Unable to construct url because [customer] field(s) are all null",
+          null,
+          null,
+          null,
+          0,
+          null);
+    }
+    return request(ApiResource.RequestMethod.POST, url, params, BankAccount.class, options);
```

Comment:
14) Update (custom): Instance method to update sub-resource having business-logic impact besides mutating the data. Same behavior of relying on instance ID attributes of instance and its parent resource just just like the standard update method on sub-resource.

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/BankAccount.java#L279-L300